.eslintcache

```diff
@@ -0,0 +1 @@
rity":1,"message":"201","line":69,"column":8,"nodeType":"194","endLine":69,"endColumn":89},{"ruleId":"200","severity":1,"message":"201","line":77,"column":8,"nodeType":"194","endLine":77,"endColumn":104},{"ruleId":"179","severity":1,"message":"202","line":1,"column":27,"nodeType":"181","messageId":"182","endLine":1,"endColumn":36},{"ruleId":"203","severity":1,"message":"204","line":27,"column":15,"nodeType":"205","endLine":27,"endColumn":35},{"ruleId":"203","severity":1,"message":"204","line":57,"column":19,"nodeType":"205","endLine":57,"endColumn":44},{"ruleId":"203","severity":1,"message":"204","line":75,"column":19,"nodeType":"205","endLine":75,"endColumn":44},{"ruleId":"179","severity":1,"message":"206","line":2,"column":23,"nodeType":"181","messageId":"182","endLine":2,"endColumn":27},{"ruleId":"183","severity":1,"message":"184","line":1,"column":1,"nodeType":"185","messageId":"186","fix":"207"},"no-native-reassign",["208"],"no-negated-in-lhs",["209"],"no-unused-vars","'educationInfo' is defined but never used.","Identifier","unusedVar","unicode-bom","Unexpected Unicode BOM (Byte Order Mark).","Program","unexpected",{"range":"210","text":"211"},"react-hooks/exhaustive-deps","React Hook useEffect has a missing dependency: 'getRepoData'. Either include it or remove the dependency array.","ArrayExpression",["212"],"jsx-a11y/iframe-has-title","<iframe> elements must have a unique title property.","JSXOpeningElement","React Hook useEffect has a missing dependency: 'getProfileData'. Either include it or remove the dependency array.",["213"],"jsx-a11y/anchor-is-valid","The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md","The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md","react/jsx-no-target-blank","Using target=\"_blank\" without rel=\"noreferrer\" is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener","'useEffect' is defined but never used.","react/jsx-no-duplicate-props","No duplicate props allowed","JSXAttribute","'Zoom' is defined but never used.",{"range":"214","text":"211"},"no-global-assign","no-unsafe-negation",[-1,0],"",{"desc":"215","fix":"216"},{"desc":"217","fix":"218"},[-1,0],"Update the dependencies array to be: [getRepoData]",{"range":"219","text":"220"},"Update the dependencies array to be: [getProfileData]",{"range":"221","text":"222"},[830,832],"[getRepoData]",[1571,1573],"[getProfileData]"]
```

Comment:
Can you remove this file, you might've pushed it again accidentally.

Reply:
Sure this is due to eslint extension in vscode I will remove it